vault.sh

```diff
@@ -302,31 +335,17 @@ vault_data() {
     cat $HOME/.config/libsh/hc_vaults.json
 }
 
+vault_config() {
```

Comment:
I like this idea!
Can we add a help block to this as well? Help option allows for us to loop over all functions and build docs in the future.

Reply:
Like the idea, can work up a help block.